From bbae011803a0371cf9fe2532491ef1e980c06f60 Mon Sep 17 00:00:00 2001
From: Debasish Biswas <debasishbsws.dev@gmail.com>
Date: Fri, 27 Feb 2026 10:38:05 +0530
Subject: [PATCH] Patch istio WORKSPACE to add patches to the vendored envoy dependency.

0002-envoy-use-system-yq.patch uses system yq instead of the Bazel-downloaded
yq to avoid architecture-specific download failures on amd64.

Signed-off-by: Debasish Biswas <debasishbsws.dev@gmail.com>
---
 WORKSPACE | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WORKSPACE b/WORKSPACE
index 68527aa..86600d4 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -50,6 +50,7 @@
         "@io_istio_proxy//bazel:0011-http-prevent-generic-upgrade-connection-poisoning.patch",
         "@io_istio_proxy//bazel:0012-Include-the-length-of-dropped-Host-headers-in-HTTP-2.patch",
         "@io_istio_proxy//bazel:0013-Fix-RBAC-bypass-via-path-parameters.patch",
+        "//:0002-envoy-use-system-yq.patch",
     ],
     sha256 = ENVOY_SHA256,
     strip_prefix = ENVOY_REPO + "-" + ENVOY_SHA,
